test/install/test_install.py

```diff
@@ -0,0 +1,234 @@
+import os
+import pytest
+import subprocess
+from textwrap import dedent
+
+from ..orchestra_shim import OrchestraShim
+from ..utils import load_json
+
+
+def check_component_A_installed_properly(orchestra, metadata_overrides={}):
+    # Test file list in root
+    expected_file_list = dedent(
+        """
+    .
+    ./bin/
+    ./include/
+    ./lib -> lib64/
+    ./lib64/
+    ./lib64/include/
+    ./lib64/pkgconfig/
+    ./libexec/
+    ./share/
+    ./share/doc/
+    ./share/man/
+    ./share/orchestra/
+    ./share/orchestra/component_A.idx
+    ./share/orchestra/component_A.json
+    ./some_file
+    ./usr/
+    ./usr/include/
+    ./usr/lib/
```

Comment:
I see at least three very similar strings such as this one.
Can we have a function that generates them, taking in input the information required to make up for the various differences?

Reply:
Now all but one file lists are factored (the one exception being the file list of a binary archive for which factoring was not worth the effort IMO)